news/_posts/core-weekly/2019-08-01-coreweekly-week-30-2019.md

```diff
@@ -0,0 +1,166 @@
+- [51 pull requests have been opened](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+created%3A2019-07-22..2019-07-28) in the project repositories;
+- [54 pull requests have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+closed%3A2019-07-22..2019-07-28), including [40 merged pull requests](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+merged%3A2019-07-22..2019-07-28).
+
+
+
+## Code changes in the 'develop' branch
+
+### Core
+
+* [#13712](https://github.com/PrestaShop/PrestaShop/pull/13712): Prepares Command API for migrating Order view page. Thank you [@sarjon](https://github.com/sarjon)
+* [#13906](https://github.com/PrestaShop/PrestaShop/pull/13906): Update CLDR data to 35.1, by [@eternoendless](https://github.com/eternoendless)
+* [#14065](https://github.com/PrestaShop/PrestaShop/pull/14065): Migrates Shopping Carts view page. Thank you [@sarjon](https://github.com/sarjon)
+
+
+### Back office
+
+* [#12497](https://github.com/PrestaShop/PrestaShop/pull/12497): Fix issue with the link Manage tax rules in the pricing tab, by [@khouloudbelguith](https://github.com/khouloudbelguith)
+* [#12903](https://github.com/PrestaShop/PrestaShop/pull/12903): Match multiline translation expressions in modules. Thank you [@unlocomqx](https://github.com/unlocomqx)
+* [#13980](https://github.com/PrestaShop/PrestaShop/pull/13980): Execute displayAdminLogin hook without shop context. Thank you [@idnovate](https://github.com/idnovate)
+* [#14805](https://github.com/PrestaShop/PrestaShop/pull/14805): Fix unclosed div tag. Thank you [@sarjon](https://github.com/sarjon)
+
+
+### Front office
+
+* [#14819](https://github.com/PrestaShop/PrestaShop/pull/14819): Fix create account sitemap link. Thank you [@Thomas-lhuillier](https://github.com/Thomas-lhuillier)
+
+
+### Tests
+
+* [#14670](https://github.com/PrestaShop/PrestaShop/pull/14670): Add Behat tests for Customer commands. Thank you [@sarjon](https://github.com/sarjon)
+* [#14764](https://github.com/PrestaShop/PrestaShop/pull/14764): Install prestashop test with puppeteer, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#14815](https://github.com/PrestaShop/PrestaShop/pull/14815): Add a new smoke test: crawling BO, by [@SimonGrn](https://github.com/SimonGrn)
+* [#14825](https://github.com/PrestaShop/PrestaShop/pull/14825): Fix some minor problems on the "crawling BO" smoke test, by [@SimonGrn](https://github.com/SimonGrn)
+
+
+### Install
+
+* [#14718](https://github.com/PrestaShop/PrestaShop/pull/14718): Remove codacy badge as we dont use it anymore, by [@matks](https://github.com/matks)
+
+
+## Code changes in the "1.7.6.x" branch (for v1.7.6.0)
```

Comment:
`## Code changes in the "1.7.6.x" branch (for v1.7.6.0)`
=>
`## Code changes in the "1.7.6.x" branch (for v1.7.6.1)` 😉@ttoine

Reply:
ok